app/boards/shields/cradio/cradio.keymap

```diff
@@ -0,0 +1,18 @@
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+
+
+/ {
+        keymap {
+                compatible = "zmk,keymap";
+
+                default_layer {
+		bindings = <
+		&kp Q &kp W &kp E &kp R &kp T 		&kp Y &kp U  &kp I    &kp O   &kp P 
```

Comment:
Seems like a "too simple" stock keymap. Obviously, a board like this is likely to be customized pretty specifically by each user, so not a huge deal if we just leave it as is.

Reply:
i would like to add a better example keymap, but it's going to rely heavily on mod taps and maybe chords, i'd like to modify it as these become more established and stabilised. (i'm using okke's branch)

Reply:
Ok, that's fine, and makes sense.

Beyond that, my only concern is the lack of the `&bt` behavior bindings anywhere, so users can reset the bond, use different profiles, etc.

Thoughts on at least adding a second layer that includes those OOTB?